### ISSUE TITLE
Bowtie 1.2 tutorial "No such file or directory"

### ISSUE THREAD

**Issue by benjab01** [BenLangmead/bowtie#36]
Attempting to run the Bowtie 1.2 tutorial, I run into an early issue. 

> fisherlabs-imac:bowtie-1.2 fisherlab$ bowtie e_coli reads/e_coli_1000.fq
> Traceback (most recent call last):
>   File "/Users/fisherlab/Desktop/bowtie-1.2/bowtie", line 91, in <module>
>     main()
>   File "/Users/fisherlab/Desktop/bowtie-1.2/bowtie", line 88, in main
>     os.execv(bin_spec, bowtie_args)
> OSError: [Errno 2] No such file or directory

I believe my path is correct. I experience this issue whether or not I use ./ before the bowtie command. I may just be inexperienced, but it would be great to have this issue clarified. If there is a bug, I hope it can be corrected!

**Comment by ch4rr0** [comment]
Did you download the source or binary package? If you retrieved the former you would need to first build bowtie by running the make allall command. Let me know if that helps.

**Comment by MelkWold** [comment]
Hi all,
I have the same problem. I am using the cmd on mac os (OS X 10.10.5, intel Core i3, iMax11.2). I downloaded the binary package and used "make" to build bowtie. I got some error messages and warnings during the process including the following:
./threading.h:10:11: fatal error: 'tbb/mutex.h' file not found
SeqAn-1.1/seqan/basic/basic_holder.h:1001:59: warning: unused typedef 'THolderType' [-Wunused-local-typedef]
        typedef typename Value<Holder<TValue, Tristate2> >::Type THolderType;
                                                                 ^
Any help is highly appreciated !
